src/targets/gpu/include/migraphx/gpu/convolution.hpp

```diff
@@ -83,9 +83,10 @@ struct miopen_convolution
 
     inline shape compute_shape(const std::vector<shape>& inputs) const
     {
-        check_shapes{inputs, op}.has(4).standard();
```

Comment:
This is intentional correct? We can't assume a standard shape since we need to do a transpose to get things in the nhwc format via our permutations?